src/curvefit/solvers/solvers.py

```diff
@@ -50,6 +50,9 @@ def fit(self, data, x_init=None, options=None):
         self.x_opt = result.x
         self.fun_val_opt = result.fun
 
+    def predict(self, **kwargs):
+        return self.model.predict(self.x_opt, **kwargs)
+
 
 class Composite(Base):
```

Comment:
Can we rename to `CompositeModel` to distinguish between the other `Composite` solver?

Reply:
Yes, for sure!